File: src/main/java/frc/robot/commands/Intake_Commands/ManualArm.java
package frc.robot.commands.Intake_Commands;

import edu.wpi.first.wpilibj2.command.Command;
import frc.robot.subsystems.Intake;

public class ManualArm extends Command {

    private double armVel;
    private Intake intake;

    public ManualArm(Intake intake, double armVel) {
        this.armVel = armVel;
        this.intake = intake;
        addRequirements(intake);
    }

    @Override
    public void initialize() {
        intake.getRaiseLowerSpinner().set(armVel);
    }

    @Override
    public void execute() {
        super.execute();
    }

    @Override
    public boolean isFinished() {
        return false;
    }

    @Override
    public void end(boolean interrupted) {
        intake.getRaiseLowerSpinner().set(0.0);
    }
}

File: src/main/java/frc/robot/commands/Intake_Commands/ManualArm.kt
package frc.robot.commands.Intake_Commands

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.robot.subsystems.Intake

class ManualArm(private val intake: Intake, private val armVel: Double) : CommandBase() {

    init {
        // each subsystem used by the command must be passed into the addRequirements() method
        addRequirements(intake)
    }

    override fun initialize() { intake.raiseLowerSpinner.set(armVel) }

    override fun execute() {}

    override fun isFinished(): Boolean { return false }

    override fun end(interrupted: Boolean) { intake.raiseLowerSpinner.set(0.0) }
}
